sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -751,6 +751,20 @@ object SQLConf {
     .checkValues(HiveCaseSensitiveInferenceMode.values.map(_.toString))
     .createWithDefault(HiveCaseSensitiveInferenceMode.NEVER_INFER.toString)
 
+  val HIVE_BLOBSTORE_SUPPORTED_SCHEMES =
+    buildConf("spark.sql.hive.blobstore.supported.schemes")
+      .doc("Comma-separated list of supported blobstore schemes.")
```

Comment:
Where does this feature to change the staging directory to, HDFS? Can you fix the documentation to describe what the configuration does? Seems like it writes the data first in staging dir, and move it to S3 because renaming on S3 is expensive.

Reply:
Thank you for the comment. Current explanation is the same sentence as Hive's. I will add explanation like `If you disable this parameter, Spark writes the data first in scratch dir, and move it to blobstore because moving it on blobstore is expensive.` based on your comment.

Reply:
Modified.